CFFI python tests miss the events by creating event emitter too late

Here is a failure from CI:
```
=================================== FAILURES ===================================
______________________ test_markseen_invalid_message_ids _______________________
[gw1] linux -- Python 3.10.16 /home/runner/work/core/core/python/.tox/py/bin/python

acfactory = <deltachat.testplugin.ACFactory object at 0x00007f8c0c0ac560>

    def test_markseen_invalid_message_ids(acfactory):
        ac1 = acfactory.get_pseudo_configured_account()
        ac2 = acfactory.get_pseudo_configured_account()
        contact1 = ac1.create_contact(ac2)
        chat = contact1.create_chat()
        chat.send_text("one message")
>       ac1._evtracker.get_matching("DC_EVENT_MSGS_CHANGED")

tests/test_4_lowlevel.py:159: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
.tox/py/lib/pypy3.10/site-packages/deltachat/events.py:118: in get_matching
    for ev in self.iter_events(timeout=timeout, check_error=check_error):
.tox/py/lib/pypy3.10/site-packages/deltachat/events.py:114: in iter_events
    yield self.get(timeout=timeout, check_error=check_error)
.tox/py/lib/pypy3.10/site-packages/deltachat/events.py:107: in get
    ev = self._event_queue.get(timeout=timeout)
/opt/hostedtoolcache/PyPy/3.10.16/x64/lib/pypy3.10/queue.py:171: in get
    self.not_empty.wait()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Condition(<unlocked _thread.lock object at 0x00007f8c0c101e80>, 0)>
timeout = None

    def wait(self, timeout=None):
        """Wait until notified or until a timeout occurs.
    
        If the calling thread has not acquired the lock when this method is
        called, a RuntimeError is raised.
    
        This method releases the underlying lock, and then blocks until it is
        awakened by a notify() or notify_all() call for the same condition
        variable in another thread, or until the optional timeout occurs. Once
        awakened or timed out, it re-acquires the lock and returns.
    
        When the timeout argument is present and not None, it should be a
        floating point number specifying a timeout for the operation in seconds
        (or fractions thereof).
    
        When the underlying lock is an RLock, it is not released using its
        release() method, since this may not actually unlock the lock when it
        was acquired multiple times recursively. Instead, an internal interface
        of the RLock class is used, which really unlocks it even when it has
        been recursively acquired several times. Another internal interface is
        then used to restore the recursion level when the lock is reacquired.
    
        """
        if not self._is_owned():
            raise RuntimeError("cannot wait on un-acquired lock")
        waiter = _allocate_lock()
        waiter.acquire()
        self._waiters.append(waiter)
        saved_state = self._release_save()
        gotit = False
        try:    # restore state no matter what (e.g., KeyboardInterrupt)
            if timeout is None:
>               waiter.acquire()
E               Failed: Timeout (>300.0s) from pytest-timeout.

/opt/hostedtoolcache/PyPy/3.10.16/x64/lib/pypy3.10/threading.py:320: Failed
----------------------------- Captured stdout call -----------------------------
0.13 [events-ac2] DC_EVENT_ACCOUNTS_ITEM_CHANGED data1=0 data2=0
0.13 [events-ac2] INFO src/key.rs:133: Converting secret key to public key.
0.42 [events-ac1] EVENT THREAD START
+++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
```

Event should definitely be there, but event thread called `dc_get_event_emitter` after the start of the test, so the event was lost.

--- a/python/src/deltachat/events.py
+++ b/python/src/deltachat/events.py
@@ -226,6 +226,10 @@ class EventThread(threading.Thread):
 
     def __init__(self, account: Account) -> None:
         self.account = account
+        self.event_emitter = ffi.gc(
+            lib.dc_get_event_emitter(self.account._dc_context),
+            lib.dc_event_emitter_unref,
+        )
         super(EventThread, self).__init__(name="events")
         self.daemon = True
         self._marked_for_shutdown = False
@@ -250,13 +254,9 @@ class EventThread(threading.Thread):
     def run(self) -> None:
         """get and run events until shutdown."""
         with self.log_execution("EVENT THREAD"):
-            event_emitter = ffi.gc(
-                lib.dc_get_event_emitter(self.account._dc_context),
-                lib.dc_event_emitter_unref,
-            )
             while not self._marked_for_shutdown:
                 with self.swallow_and_log_exception("Unexpected error in event thread"):
-                    event = lib.dc_get_next_event(event_emitter)
+                    event = lib.dc_get_next_event(self.event_emitter)
                     if event == ffi.NULL or self._marked_for_shutdown:
                         break
                     self._process_event(event)

